recipe/meta.yaml

```diff
@@ -115,16 +110,15 @@ outputs:
 about:
   home: https://matplotlib.org/
   license: LicenseRef-PSF-based
-  license_url: https://matplotlib.sourceforge.net/users/license.html
+  license_url: https://matplotlib.org/stable/users/project/license.html
   license_family: PSF
   license_file: LICENSE/LICENSE
   summary: Publication quality figures in Python
   description: |
-    matplotlib is a python 2D plotting library which produces publication
-    quality figures in a variety of hardcopy formats and interactive
-    environments across platforms. matplotlib can be used in Python scripts,
-    the Python and IPython shell s(ala MATLAB or Mathematica), web
-    application servers, and six graphical user interface toolkits.
+    Matplotlib produces publication-quality figures in a variety of hardcopy formats
+    and interactive environments across platforms. Matplotlib can be used in
+    Python scripts, Python/IPython shells, web application servers,
+    and various graphical user interface toolkits.
   dev_url: https://github.com/matplotlib/matplotlib/
   doc_url: https://matplotlib.org/contents.html
```

Comment:
The up-to-date url seems to be this one: https://matplotlib.org/stable/users/index

Reply:
Fixed.